### PR TITLE
api: add os.realpath

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

**Comment by jazzdan** [comment]
@nicks for our next trick, do you know how to get nested submodules to work with the docs generator? What I tried in the most recent commit here _doesn't_ work. Any ideas?

**Comment by maiamcc** [comment]
I can maybe help but i'm not sure what you're shooting for--can you explain more?

**Comment by jazzdan** [comment]
@maiamcc Hey, yeah like right now we have `api/modules/os.py` for documenting things in the `os` module. But now that we've added the `os.path` module how do I represent that in this sphinx thing?

`api/modules/os/path.py` didn't work

File: api/modules/os/path.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def realpath(path: str) -> str:
```
Comment (nicks) [inline api/modules/os/path.py]: wait...do you need to regenerate the html?

Comment (jazzdan) [reply api/modules/os/path.py]: That should happen when you run `tilt up`, right?

**Pull request closed without merging**